shell/platform/android/android_surface_vulkan_impeller.cc

```diff
@@ -56,6 +56,9 @@ AndroidSurfaceVulkanImpeller::AndroidSurfaceVulkanImpeller(
       CreateImpellerContext(proc_table_, workers_, enable_vulkan_validation);
   is_valid_ =
       proc_table_->HasAcquiredMandatoryProcAddresses() && impeller_context_;
+  if (is_valid_) {
+    FML_LOG(ERROR) << "Using the Impeller Vulkan rendering backend.";
```

Comment:
Perhaps just an info log. Not really an error that we end up using Vulkan.

Reply:
This matches the behavior on the GL side, and I assume it was done this way because info logs by default do not show up from `flutter run`.